RFCs/FS-1004-result-type.md

```diff
@@ -48,3 +48,42 @@ The standard range of testing considerations for F# library types apply.
 - declare as struct
 - compiled name
 - require additional functions?
+
+### Proposal for additional functions
+
+Minimal:
+
+```fsharp
+bind : ('TSuccess -> Result<'T, 'TError>) -> Result<'TSuccess, 'TError> -> Result<'T, 'TError>
```

Comment:
I prefer short type variable names (`Result<'a,'e>`), but was skimming MSDN and trying to follow conventions there. (I obviously inferred them incorrectly :) ).